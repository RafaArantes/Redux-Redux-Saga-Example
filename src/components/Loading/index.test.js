import React from 'react';
import {configure, shallow} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';
import {Loading} from './index';
import {connect} from 'react-redux'


configure({ adapter: new Adapter() });

describe(`<Loading>`, () =>{
  it('renders three <Loading /> components', () => {
    const loading = shallow(<Loading loading={true}/>);
    expect(loading).toMatchSnapshot()
  });
})
