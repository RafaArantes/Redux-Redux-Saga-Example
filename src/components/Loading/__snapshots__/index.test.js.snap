// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading> renders three <Loading /> components 1`] = `ShallowWrapper {}`;
